src/rcms/fm/app/level1/HCALStateNotificationHandler.java

```diff
@@ -56,9 +56,6 @@ public void processNotice( Object notice ) throws UserActionException {
       //    "from this state: " + notification.getFromState()  +
       //    " to this state: " + notification.getToState());
       
-      String actualState = fm.getState().getStateString();
-      //logger.warn("["+fm.FMname+"]: FM is in state: "+actualState);
-
```

Comment:
These two lines will be useful for stateMachine debugging. I think we should leave them in place.

Reply:
OK, I will restore these but move them to "info" level.

Reply:
Actually looking at this, the `logger.warn` was already commented out. Do you want it back?